pkg/command/context.go

```diff
@@ -976,8 +976,9 @@ func listDeactivatedPlugins(deactivatedPlugins []discovery.Discovered, ctxName s
 }
 
 func displayContextListOutputListView(cfg *configtypes.ClientConfig, writer io.Writer) {
+	fmt.Printf("Prem targetStr is %v\n", targetStr)
```

Comment:
To remove 😄

Reply:
Thank you for catching this 😄 . Removed the debug print